README.md

```diff
@@ -1081,7 +1081,15 @@ Connect server, as well as the language (`-l, --language`), version (`-V,
 delete. Deleting a large cache might take a while. The command will wait for
 Connect to finish the task.
 
-Use the dry run option (`-d, --dry-run`) to surface any errors ahead of
+Use the following parameters specify the target cache:
+
+- `language` (required) must name `R` or `Python`. It is case-insensitive.
+- `version` (required) must be a three-part version number, e.g. `3.8.12`.
+- `image-name` (optional) defaults to `Local`, which targets caches used for
+  natively-executed content. Off-host images can be specified using either the
```

Comment:
Always using four-space indentation is a good way to make sure that the different processors render the Markdown consistently.

Reply:
In this case, it was just that the list was not in its own paragraph, i.e. not separated from the previous line by an empty line. I don't think indentation would do anything here? Anyway, it seems to be rendering correctly, so I'm going to resolve this conversation.